pages/Favorites/components/FavoritesList/components/Item/components/CTAButtons/index.jsx

```diff
@@ -44,7 +44,8 @@ const CTAButtons = props => (
       className={styles.cartButton}
       handleAddToCart={() => handleAddToCart(props)}
       isLoading={false}
-      isOrderable={!props.isBaseProduct(props.productId)}
+      isDisabled={!props.isOrderable(props.productId)}
```

Comment:
This will change with the mentioned changes in the connector:
```
isDisabled={!props.isOrderable}
isOrderable={!props.isBaseProduct && props.isOrderable}
```

Reply:
Done.